feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/SearchActivity.kt

```diff
@@ -68,10 +78,12 @@ class SearchActivity : ComponentActivity() {
               selectClaim = viewModel::onSelectClaim,
               commonClaims = viewState.commonClaims,
               selectOther = {
-                startClaimsFlow(
-                  viewModel = viewModel,
-                  commonClaimId = null,
-                )
+                lifecycleScope.launch {
+                  startClaimsFlow(
+                    viewModel = viewModel,
+                    commonClaimId = null,
+                  )
+                }
```

Comment:
I think since we're inside compose right here, it's preferable to use the coroutine context provided through `rememberCoroutineScope`.
By that I mean basically in line 76, add `val coroutineScope = rememberCoroutineScope()`, and inside here, instead of doing `lifecycleScope.launch {` you can do `coroutineScope.launch {`.
This means that it will both follow the composable's lifecycle better, but also if we ever extract this function outside of the activity (where we're getting the lifecycleScope from), then it'd still work.